UI/MapboxNavigationUI/Bundle.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+extension Bundle {
+    class var navigationUI: Bundle {
+        get {
+            let bundle = Bundle(for: RouteViewController.self)
+            let resourceBundlePath = "\(bundle.bundlePath)/MapboxNavigationUI.bundle"
```

Comment:
> We strongly recommend library developers to adopt resource bundles as there can be name collisions using the resources attribute.

I believe this guidance is only applicable to static libraries, which is the default format for CocoaPods.